app-frontend/src/app/pages/login/login.module.js

```diff
@@ -16,6 +16,7 @@ class LoginController {
         if (this.authService.verifyAuthCache()) {
             this.$state.go('home');
         } else {
+            this.authService.clearAuthStorage();
```

Comment:
probably want to do this every place we currently delete credentials, instead of just here. like during logout.